### PR TITLE
fluxible-addons-react: add migration guide documentation

### PR DESCRIPTION
**Pull request by pablopalacios**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/fluxible-addons-react/MIGRATING.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## From 0.2.x to 1.0.0
+
+### provideContext
+
+`provideContext(Component, customContextTypes)` -> `provideContext(Component, plugins)`
+
+Since the new React API doesn't rely on PropTypes anymore, a list of
+plugins keys is enough to tell fluxible which custom data you would
+like to have avaialble inside fluxible context.
```
Comment (redonkulus) [inline packages/fluxible-addons-react/MIGRATING.md:11]:
```suggestion
like to have available inside fluxible context.
```

File: packages/fluxible-addons-react/MIGRATING.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+related to fluxible itself, you will need to provide your own
+solution to achieve the same result as before.
+
+### connectToStores
+
+`connectToStores(Component, stores, getStateFromStores, customContextTypes)` -> `connectToStores(Component, stores, getStateFromStores)`
+
+Since the new React API doesn't rely on PropTypes anymore, there is no
+need to specify customContextTypes to extract from the fluxible
+context. The context avaialble to your `getStateFromStores` will
```
Comment (redonkulus) [inline packages/fluxible-addons-react/MIGRATING.md:42]:
```suggestion
context. The context available to your `getStateFromStores` will
```

**Comment by pablopalacios** [comment]
@redonkulus I've renamed the file to `UPGRADE.md` to be consistent with `fluxible-router`. Besides that, do you have any further comments here?

**Review by redonkulus — APPROVED** [review]
Nope, this looks good to me! Thanks. Lets keep updating this in new PRs that make breaking changes.

**Pull request merged**